tests/fixture/mig_shared_configuration--new.json.ini

```diff
@@ -0,0 +1,6 @@
+[DEFAULT]
```

Comment:
Not sure if the double hyphens in the filename are on purpose so I'll keep them for now.

Reply:
They were intentional - the idea was you'd have a series of fixtures that all correspond to the behaviour of a particular file and you'd some indication of that. The double hyphen was lifted from BEM where it is used for specifying "variations".